pktables.py

```diff
@@ -9,7 +9,7 @@
 
 
 PACKETKEY = os.environ.get("PACKET_API_AUTH_TOKEN", os.environ.get("PACKETKEY"))
-PROJECTID = os.environ.get("PACKET_PROJECT", os.environ.get("PROJECTID"))
+PROJECTID = os.environ.get("PACKET_PROJECT_ID", os.environ.get("PROJECTID"))
```

Comment:
how about
```suggestion
PROJECTID = os.environ.get("PACKET_PROJECT_ID", os.environ.get("PACKET_PROJECT", os.environ.get("PROJECTID")))
```
instead? Otherwise just drop the compatibility and do just `PACKET_PROJECT_ID`